packages/shared/locales/en.json

```diff
@@ -1184,7 +1184,8 @@
             "cannotUseAccount": "Unable to use the specified account.",
             "cannotFindStakingEvent": "Unable to find staking event.",
             "cannotVisitAirdropWebsite": "Unable to open the {airdrop} website.",
-            "invalidStakingEventId": "Invalid staking event ID."
+            "invalidStakingEventId": "Invalid staking event ID.",
+            "errorOccurred": "An error occurred while trying to {action} your funds."
```

Comment:
just a mini thing, but I think `{action}` will not play well with translations

Reply:
Good catch. This is actually no longer needed.